articles/storage/common/storage-use-azcopy-troubleshoot.md

```diff
@@ -24,6 +24,8 @@ If the exit code is `1-error`, then examine the log file. Once you understand th
 
 If the exit code is `2-panic`, then check the log file exists. If the file doesn't exist, file a bug or reach out to support.
 
+If the exit code is any other non-zero exit code, it may be a exit code from the system (e.g. OOM kill). Check your operating system documentation for special exit codes.
```

Comment:
```suggestion
If the exit code is any other non-zero exit code, it may be an exit code from the system. For example, OOMKilled.  Check your operating system documentation for special exit codes.
```
'eg.' isn't supposed to be used in Docs.

Reply:
For the sake of learning, what's the exact reasoning behind this?

Reply:
@adreed-msft - I'm not sure of the reasoning, but my guess is because it's less clear and doesn't localize very well.  Here's the MS Style Guide entry: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33667